src/pages/mathematics/e-eulers-number/index.md

```diff
@@ -12,4 +12,14 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 #### More Information:
 <!-- Please add any articles you think might be helpful to read before writing the article -->
 
+Euler's Number is a famous irrational mathematical constant that serves as the base of the natural logarithm. It is named after the Swiss mathematician, Leonhard Euler, and it can be identified with the following symbol, &#8455;. This number has many important uses and roles in mathematics and is applied in calculating compound interest, for example. You would use Euler's Number in calculus in order to be able to perform differential and integral calculus with exponential functions and logarithms.
 
+![Leonhard_Euler Image](https://github.com/TomerPacific/fccGuideImages/blob/master/Leonhard_Euler.jpg?raw=true)
+
```

Comment:
Please check this whitespace that is being created here. I'm not sure what is being done?
You can see it if you go to [this url](https://github.com/TomerPacific/guides/blob/50097d4c9a7b21c16720a6e170d67efe3b4b5a82/src/pages/mathematics/e-eulers-number/index.md)

Reply:
Edited the whitespace. It happened because the unicode for Euler's number did not render.

Reply:
It's still not rendering. Did you check [here](https://github.com/TomerPacific/guides/blob/patch-5/src/pages/mathematics/e-eulers-number/index.md) to see if it has been fixed?